### ISSUE TITLE
http://devops/test.json doesn't exist?

### ISSUE THREAD

**Issue by bootstraponline** [errm/http_shooting_party#1]
Is there a local server that should serve `http://devops/test.json`? I'm trying to run the benchmark and it fails due to 404.

**Comment by bootstraponline** [comment]
If I modify [test.rb#L10](https://github.com/errm/http_shooting_party/blob/f039ef708ee168a06bf320ddd47d911afb234b8f/lib/http_shooting_party/test.rb#L10) to point to `test.json` with the content `{"foo":"bar"}` served by passenger then the csv file is correctly generated.

**Comment by errm** [comment]
Yeah, that was a server in my office. Perhaps the script should start a server also.
